src/doc/rustdoc/src/how-to-read-rustdoc.md

```diff
@@ -0,0 +1,105 @@
+# How to read rustdoc output
+
+Rustdoc's HTML output includes a friendly and useful navigation interface which 
+makes it easier for users to navigate and understand your code.
+This chapter covers the major features of that interface,
+and is a great starting point for documentation authors and users alike.
+
+## Structure
+
+The `rustdoc` output is divided into three sections.
+Along the left side of each page is a quick navigation bar,
+which shows contextual information about the current entry.
+The rest of the page is taken up by the search interface at the top
+and the documentation for the current item below that.
+
+## The Item Documentation
+
+The majority of the screen is taken up with the documentation text for the item
+currently being viewed.
+At the top is some at-a-glance info and controls:
+
+- the type and name of the item,
+  such as "Struct `std::time::Duration`",
+- a button to copy the item's path to the clipboard,
+  which is a clipboard item
+- a button to collapse or expand the top-level documentation for that item
+  (`[+]` or `[-]`),
+- a link to the source code (`[src]`),
+  if [configured](the-doc-attribute.html#html_no_source),
```

Comment:
```suggestion
  if [configured](the-doc-attribute.html#html_no_source) and present. The source may not be available if the documentation was created with `cargo doc --no-deps`.
```

Reply:
Hi @NoraCodes - did you see this comment? Was it intentional to not add this sentence?

Reply:
Yeah, sorry! I thought there was a discussion about this but now I can't find it - did I hallucinate that? lol

Reply:
Okay, yeah, still can't find it. I don't think we should mention this here because it's intended to be a guide for docs readers - we document `--no-deps` elsewhere.

Reply:
Hmm, so the reason I thought to mention it was because it's pretty common for people to be confused by reading docs and having missing links. But I don't feel super strongly about it.

Reply:
> Yeah, sorry! I thought there was a discussion about this but now I can't find it - did I hallucinate that? lol

@NoraCodes I think you're thinking of https://github.com/rust-lang/rust/pull/90487#discussion_r741328790, which was a discussion about another command-line flag.